packages/cli/src/utils/generate.ts

```diff
@@ -165,13 +165,6 @@ async function copyStoreConfig() {
 }
 
 function mergeCMSFiles() {
-  try {
-    mkdirsSync(`${tmpDir}/cms`)
-    console.log(`${chalk.green('success')} - CMS folder created`)
-  } catch (err) {
-    console.error(`${chalk.red('error')} - ${err}`)
-  }
-
```

Comment:
this code is unnecessary because we always copy the entire @faststore/core with each modification, so there will always be the cms folder.